.gitlab-ci.yml

```diff
@@ -0,0 +1,28 @@
+image: node:10
+
+stages:
+  - test_ci
+  - release
+
+cache:
+  key: ${CI_COMMIT_REF_SLUG}
+  paths:
+    - node_modules/
+  policy: pull
```

Comment:
I am not sure about `pull` only. Does it mean that you will not push `node_modules` after the install? Then I don't see the point of this cache. 🙂